components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarPresenter.kt

```diff
@@ -64,16 +65,39 @@ class ToolbarPresenter(
         val session = sessionId?.let { sessionManager.findSessionById(sessionId) }
             ?: sessionManager.selectedSession
 
-        renderer.post(session?.url ?: "")
+        val urlToPost = if (session != null) {
+            getURLToPost(session, session.url)
+        } else {
+            ""
+        }
+
+        // Set the title to be the url as we don't have a title until the page loads
+        session?.let { onTitleChanged(it, it.url) }
+
+        renderer.post(urlToPost)
```

Comment:
This logic shouldn't live here. The consuming app can already provide a `UrlRenderConfiguration` to define how the URL should be rendered. If we want a "host" only mode (Btw. is it really the host? Or the "registrable domain"?).

Reply:
Our [mocks](https://github.com/mozilla-mobile/fenix/issues/203#issuecomment-463426620) used the word "host" and that seemed to give me the right string (aka "google.com"). How could I get the "registrable domain"?

Reply:
See `UrlRenderer` this is already used when coloring the domain. That's also a good example - if you look at desktop Firefox, the highlighted part of the URL is the "registrable domain".

Reply:
It looks like `CustomToolbarFeature` uses the same `oolbarPresenter` as `ToolbarFeature`. Because of this, I'm struggling to use a config that displays only the registrable domain. When the config is set to `showOnlyRegistrableDomain` for example, it will show in both custom and normal tabs, which is not the behavior we want.

Any recommendations on a clean way to solve this?